From 7b2a3234b27b3dbfe536d49ac28d5a74f2c5deec Mon Sep 17 00:00:00 2001
From: David Rientjes <rientjes@google.com>
Date: Thu, 16 Aug 2012 12:25:31 -0700
Subject: [PATCH 255/507] mm, slab: remove page_get_cache

page_get_cache() isn't called from anything, so remove it.

Signed-off-by: David Rientjes <rientjes@google.com>
Signed-off-by: Pekka Enberg <penberg@kernel.org>
Signed-off-by: Simarpreet Singh <simar@linux.com>
---
 mm/slab.c |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/mm/slab.c b/mm/slab.c
index cb6ce2d..3b4587b 100644
--- a/mm/slab.c
+++ b/mm/slab.c
@@ -515,12 +515,6 @@ EXPORT_SYMBOL(slab_buffer_size);
 static int slab_max_order = SLAB_MAX_ORDER_LO;
 static bool slab_max_order_set __initdata;
 
-static inline struct kmem_cache *page_get_cache(struct page *page)
-{
-	BUG_ON(!PageSlab(page));
-	return page->slab_cache;
-}
-
 static inline struct kmem_cache *virt_to_cache(const void *obj)
 {
 	struct page *page = virt_to_head_page(obj);
-- 
1.7.9.5

